### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java [hunk 1]
```diff
@@ -66,66 +61,6 @@ public SetContextSteps(IUiContext uiContext, IWebDriverProvider webDriverProvide
...
-     * @param name Any attribute or text value of the element
-     * @param state A state value of the element
-     * (<i>Possible values:</i> <b>ENABLED, DISABLED, SELECTED, NOT_SELECTED, VISIBLE, NOT_VISIBLE</b>)
-    */
-    @When("I change context to a [$state] element with the name '$name'")
-    public void changeContextToElementWithName(State state, String name)
-    {
-        resetContext();
-        WebElement element = baseValidations.assertIfElementExists(
-                String.format("An element with the name '%1$s'", name),
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java:81]: known issues might be broken here, I propose to keep the commit, but postpone it till `0.3.0`

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java [hunk 2]
```diff
@@ -66,66 +61,6 @@ public SetContextSteps(IUiContext uiContext, IWebDriverProvider webDriverProvide
...
-     * @param attributeType A type of the element's attribute
-     * @param attributeValue A value of the element's attribute
-    */
-    @When("I change context to an element with the attribute '$attributeType'='$attributeValue'")
-    public void changeContextToElementWithAttribute(String attributeType, String attributeValue)
-    {
-        resetContext();
-
-        WebElement element = baseValidations.assertIfElementExists(
-                String.format(AN_ELEMENT_WITH_THE_ATTRIBUTE, attributeType, attributeValue),
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java:99]: known issues might be broken here, I propose to keep the commit, but postpone it till `0.3.0`

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java [hunk 3]
```diff
@@ -66,66 +61,6 @@ public SetContextSteps(IUiContext uiContext, IWebDriverProvider webDriverProvide
...
-     * @param attributeType A type of the element's attribute
-     * @param attributeValue A value of the element's attribute
-     * @param state A state value of the element
-     * (<i>Possible values:</i> <b>ENABLED, DISABLED, SELECTED, NOT_SELECTED, VISIBLE, NOT_VISIBLE</b>)
-    */
-    @When("I change context to a [$state] element with the attribute '$attributeType'='$attributeValue'")
-    public void changeContextToStateElementWithAttribute(State state, String attributeType, String attributeValue)
-    {
-        resetContext();
-        WebElement element = baseValidations.assertIfElementExists(String.format(AN_ELEMENT_WITH_THE_ATTRIBUTE,
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java:118]: known issues might be broken here, I propose to keep the commit, but postpone it till `0.3.0`

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WebElementsSteps.java [hunk 4]
```diff
@@ -68,48 +68,45 @@ public void ifTextMatchesRegex(String regex)
...
-                    assertCondition = pattern.matcher(pseudoElementContent).find();
-                }
-            }
+            assertCondition = verifyText(regex, actualText, isWebElement);
         }
         softAssert.assertTrue("The text in search context matches regular expression " + regex,
                 assertCondition);
     }
 
+    private boolean verifyText(String regex, String actualText, boolean isWebElement)
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WebElementsSteps.java:77]: isWebElement -> webElement :)

**Review by uarlouski — APPROVED** [review]

**Pull request merged**